lib/extractor/default.nix

```diff
@@ -33,6 +33,11 @@ let
     pkgs.symlinkJoin {
       name = "${python_env.name}-patched";
       paths = [ python_env ];
+      buildInputs = [
+        # prefare binary wrapper - but if that's not available (e.g. nixos 21.05)
+        # use the regular shell script wrapper
+        pkgs.makeBinaryWrapper or pkgs.makeWrapper
```

Comment:
I was actually surprised this list element didn't have to be inside a parenthesis :-)

Reply:
I suppose x.y or z has actually the highest precedence in nix: https://nixos.org/manual/nix/unstable/expressions/language-operators.html ;)